src/app/ui/earn/EarnFilteringContext.tsx

```diff
@@ -69,6 +79,9 @@ export const EarnFilteringProvider = ({
   const address: Hex | undefined = useAccountAddress();
   const usedYourAddress = address !== undefined;
 
+  // TODO: introduce the loading state?
+  const [sortBy, setSortBy] = useState<SortByOptions>(SortByOptions.APY);
```

Comment:
For what it's worth, in https://github.com/jumperexchange/jumper-backend/pull/482
I add sorting there: https://github.com/jumperexchange/jumper-backend/blob/b874e5a11cfaa63ca97739eaed7f182c0a6f25a9/src/earn/earn.dto.ts#L168-L202